src/components/Cloud/Pull-Requests.js

```diff
@@ -12,15 +12,7 @@ import PullRequestItem from './Pull-Request-Item';
 const PullRequests = ( { items } ) => <DashboardBlock title="Pull Requests">
 	{ ( items && items.length > 0 )
 		? <ul>
-			{ items.map( pr => <PullRequestItem
-				date={ pr.date }
-				id={ pr.id }
-				link={ pr.link }
-				status={ pr.status }
-				statusText={ pr.statusText }
-				title={ pr.title }
-				key={ pr.id }
-			/> ) }
+			{ items.map( pr => <PullRequestItem {...pr} /> ) }
```

Comment:
I think it still needs a key. Will only throw a warning for now though so will fix later.